experimental-design/README.adoc

```diff
@@ -141,5 +141,70 @@ Some search engines as MaxQuant use the Fraction Group to perform better statist
 - Comment [Fraction Group]: Some Quantitative tools use the Fraction Group to know which fractions belong together. In MaxQuant the Fraction Group is called "Experiment".
 
 
+[[encoding-MSRun-technical-details]]
+== MSRun technical details properties
 
+We RECOMMEND to encode some of the technical parameters of the mass spectrometry experiment as Comments (https://www.ebi.ac.uk/arrayexpress/help/creating_a_sdrf.html[Check what is a Comment in SDRF]) including the following parameters:
 
+- Protein Modifications <<encoding-protein-modifications>>
+- Precursor and Fragment mass tolerances <<encoding-tolerances>>
+- Enzyme <<encoding-enzymes>>
+
+[[encoding-protein-modifications]]
+=== Protein Modifications
+
+Sample modifications (PTMs) are originated from multiple sources: **technical modifications**, **isotope labeling** or the most **biologically relevant** the wide variety of chemical modifications after translation. The most common and widely studied post translational modifications include phosphorylation and glycosylation. Many of these post-translational modifications are critical to the protein's function.
+
+The current specification RECOMMEND to provide Sample modifications including the Amino acid affected, if is Variable or Static (Fixed) and other properties such as mass shift and position (e.g. anywhere in the sequence).
+
+The RECOMMENDED name of the column for sample modification parameters is:
+
+  Commet [modification parameters]
+
+.. NOTE: The `modification parameters` is the name of the ontology term https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1001055[MS:1001055]
+
+For each modification, we will capture different properties in a `key=value` pair structure including name, position, etc. All the possible features available for modification parameters:
+
+
+|===
+|Property |Key |Example | Required/Optional |Comment
+
+|Name of the modification| NM | NM=Acetylation | Required | Name of the modification, for custom modifications can be a name defined by the user.
+
+|Database Accession| AC | AC=UNIMOD:1 | Optional | Accession in an external database UNIMOD or PSI-MOD supported.
+
+|Chemical Formula  | CF | CF=H(2)C(2)O| Optional | This is the chemical formula of the added or removed atoms. For the formula composition please follow the guidelines from http://www.unimod.org/names.html[UNIMOD]
+
+|Modification type | MT | MT=Fixed | Required | This specifies which modification group the modification should be included with. Choose from the following options: [Fixed, Variable, Custom].
+
+|Position of the modification in the polypeptide |  PP | PP=Any N-term | Required | Choose from the following options: [Anywhere, Protein N-term, Any N-term]
+
+|Target Amino acid| TA | TA=N-term | Optional | The target amino acid letter. If the modification target multiple sites, it should be provided as Target Regular Expression (TR)
```

Comment:
TA would not equal N-term. It needs to be an amino acid or motif. Some wildcards should also be allowed. 
X-for sure. As for motifs, glycans commonly show as NXT for example.
B -> D or N
J -> I or L
Z -> E or Q

Reply:
We need to discuss in details How do we specify the Amino Acid because we have two comments related with Sites: 

- @RonBeavis (https://github.com/PRIDE-Archive/pride-metadata-standard/issues/13#issuecomment-522070819) is talking about more complex use cases. 

- @mvaudel also suggested (https://github.com/PRIDE-Archive/pride-metadata-standard/issues/13#issuecomment-521939841) to use regular expressions.

Reply:
We can use `TA` for the most mentioning the Amino Acids that will be modified. For more complex rules then we can create another `key`. 

How do we represent C-term and N-term modifications? How are you doing it now @mvaudel and @trishorts @lgatto

Reply:
I like C-term or N-term for specifying modification at the terminus. We will use 'X' as the target amino acid for N-term modifcation (e.g. TMT-label).